project/Build.scala

```diff
@@ -1243,7 +1243,7 @@ object Build {
       libraryDependencies += ("org.scala-js" %%% "scalajs-dom" % "1.1.0").cross(CrossVersion.for3Use2_13)
     )
 
-  def generateDocumentation(targets: Seq[String], name: String, outDir: String, ref: String, params: Seq[String] = Nil) =
+  def generateDocumentation(targets: Seq[String], name: String, outDir: String, ref: String, params: Seq[String] = Nil, usingScript: Boolean = true) =
```

Comment:
This can be fixed later but do you know why we need usingScript in some situations and couldn't just always use Compile/run ?

Reply:
It was my idea to have some coverage for scripts. Beside that there are no blockers to switch all cases to running u`Compile/run`

Reply:
I see, for consistency I suggest always using Compile/run and instead test the scaladoc binary from https://github.com/lampepfl/dotty/blob/master/project/scripts/cmdTests

Reply:
@pikinier20 can we change that in follow up PR?

Reply:
> @pikinier20 can we change that in follow up PR?

Sure